anitya/lib/backends/maven.py

```diff
@@ -53,9 +58,18 @@ def get_versions(cls, project):
             when the versions cannot be retrieved correctly
 
         '''
-        url = 'http://repo1.maven.org/maven2/{path}'\
-              .format(path=project.name.replace('.', '/').replace(':', '/'))
-
-        regex = r'\<a[^>]+\>(\d[^</]*)'
-
-        return get_versions_by_regex(url, regex, project)
+        if re.match(r'https?://repo\d+.maven.org/', project.homepage):
+            url = project.homepage
+            coordinates = None
+        else:
+            coordinates = project.version_url or project.name
+        if coordinates:
+            if ':' not in coordinates:
+                raise AnityaPluginException(
+                    "Aritfact coordinates need to be in format "
```

Comment:
It's official term: https://maven.apache.org/pom.html#Maven_Coordinates
Do you have an idea for better wording?